dashboard-ui/src/components/AggregateResults/DataFunctions.js

```diff
@@ -429,8 +429,8 @@ function populateHumanDataRow(rowObject) {
         "ST_6.2": "",
         "ST_8.1": "",
         "ST_8.2": "",
-        "AD_KDMA_Env": rowObject[adept].data.alignment.kdma_values.length > 0 ? Math.round(rowObject[adept].data.alignment.kdma_values[0].value * 100) / 100 : "",
-        "ST_KDMA_Env": rowObject[soartech].data.alignment !== null && rowObject[soartech].data.alignment.kdma_values.length > 0 ? Math.round(rowObject[soartech].data.alignment.kdma_values[0].value * 100) / 100 : "",
+        "AD_KDMA_Env": rowObject[adept].data.alignment.kdma_values !== undefined && rowObject[adept].data.alignment.kdma_values.length > 0 ? Math.round(rowObject[adept].data.alignment.kdma_values[0].value * 100) / 100 : "",
+        "ST_KDMA_Env": rowObject[adept].data.alignment.kdma_values !== undefined ? rowObject[soartech].data.alignment !== null && rowObject[soartech].data.alignment.kdma_values.length > 0 ? Math.round(rowObject[soartech].data.alignment.kdma_values[0].value * 100) / 100 : "" : "",
```

Comment:
There's already a fix for this in development using optional chaining to catch if anything before kdma_values is undefined or null

Reply:
ah thanks, I'll revert this then